src/shared/components/GeneSelectionBox/GeneSymbolValidator.tsx

```diff
@@ -55,7 +56,7 @@ export default class GeneSymbolValidator extends React.Component<IGeneSymbolVali
 				</div>
 			);
 
-		if (this.props.genes.isPending && this.props.genes.result!.suggestions.length == 0)
+		if (this.props.genes.isPending && this.props.genes.result!.suggestions.length == 0 && !this.props.hideValidatingMessage)
```

Comment:
lets use either just the false (this.props.genes.result!.suggestions.length) or === but not double equal sign